gprofiler/utils.py

```diff
@@ -446,3 +446,8 @@ def get_kernel_release() -> Tuple[int, int]:
     """Return Linux kernel version as (major, minor) tuple."""
     major_str, minor_str = os.uname().release.split(".", maxsplit=2)[:2]
     return int(major_str), int(minor_str)
+
+
+def convert_to_proc_root_path(path: str, pid: int) -> str:
```

Comment:
Can remove the function and just use: `resolve_host_path` https://github.com/Granulate/granulate-utils/blob/master/granulate_utils/linux/ns.py#L167

Reply:
Make sure to update `granulate-utils` submodule